### PR TITLE
Concurrency / it-academy / JC1-110-22 / Evgeniy Antonovich / Done tasks 1-8 

### PR DESCRIPTION
**Pull request by aevgeni**
## Pull request state

Actual status for PR:
- [x] implemented
- [x] reviewed
- [x] fixed
- [ ] approved

### CODE REVIEW

File: src/main/java/com/rakovets/course/java/core/practice/concurrency/Consumer.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.Queue;
+
+public class Consumer implements Runnable {
+    private final int MILLIS = 1000;
+    private Queue<Integer> integerNumbersQueue;
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/concurrency/Consumer.java:11]:
```suggestion
    private final int MILLIS = 1000;
    
    private Queue<Integer> integerNumbersQueue;
```

File: src/main/java/com/rakovets/course/java/core/practice/concurrency/Producer.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        integerNumbersQueue = queueOfIntegerNumbers.getQueue();
+    }
+
+    @Override
+    public void run() {
+        while (true) {
+            Scanner scanner = new Scanner(System.in);
+            System.out.print("Enter integer number: ");
+            try {
+                if (scanner.hasNextInt()) {
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/concurrency/Producer.java:19]: What happens if the value is not a number?

**Review by rakovets — CHANGES_REQUESTED** [review]
Need to fix

File: src/main/java/com/rakovets/course/java/core/practice/concurrency/Producer.java (hunk 2 above)
Comment (aevgeni) [reply src/main/java/com/rakovets/course/java/core/practice/concurrency/Producer.java:19]: throw new UserInputException("Enter only integer number!");

**Pull request closed without merging**